dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -20,11 +20,19 @@
        Otherwise, play it safe by using a temporary table.
   #} */
 
+  {% if language == "python" and tmp_relation_type == "table" %}
```

Comment:
Is this supposed to be the opposite?
```suggestion
  {% if language == "python" and tmp_relation_type != "table" %}
```

Reply:
depends on the use case honestly, if just doing a insert then a temp view would be quicker, but if you want to do a actual delete and insert it must be a temp table. 

curious if this truly needs to be raised or if we can somehow make the use cases beyond easy to understand